src/creature.h

```diff
@@ -317,6 +318,9 @@ class Creature
 
         virtual bool in_sleep_state() const;
 
+        virtual bool stability_check(int intensity) const = 0;
+        //Returns true if the target will be moved
```

Comment:
I think you'd've been better off naming this `is_drag_movable` or `is_grab_movable`. Or inverting the boolean and naming it `has_stable_footing`. The generic name could go either way, so if someone wants to know which way it goes, they **must** look up this comment.